source/cloud/public/marketplace.md

```diff
@@ -0,0 +1,20 @@
+# eCloud Public VM Marketplace
+
+Specific OS customisation is available via the [eCloud marketplace](https://my.ukfast.co.uk/ecloud-public/marketplace). From you can chose a specific template and launch a VM pre-configured for this by selecting the template then launch appliance as shown below:
+
+![launchAppliance](files/launchAppliance.png)
+
+From here you will be taken to the below device configuration page where vCPU, RAM, Disk space and other attributes of the VM can be specified. You may also be asked for unqiue information to that template as in the below example a MySQL root password defined.
+
+![applianceConfig](files/applianceConfig.png)

```

Comment:
files for both applianceconfig.png and applianceConfig.png - are they both needed?

Reply:
Removed file not needed